src/instruction.cpp

```diff
@@ -1,3 +1,5 @@
+#include <bitset>
```

Comment:
Where does this file use bitset?

Reply:
Will check, maybe a mistake or a dependency, will check when I get home.

Reply:
Removed spurious `#include <bitset>` from instruction.cpp. Previously it was needed as originally I'd done the change on top of e14c76ff4518800d300dfacdfb7b7bbbe5c7e8d6 where it was needed. I then just rebased on top of master and now it's not needed :)